deploy/docker/auditbeat.docker.yml

```diff
@@ -0,0 +1,21 @@
+auditbeat.modules:
```

Comment:
pinging @andrewkroh on this file

Reply:
@andrewkroh I have not made any changes to the file, just copied from /beats/auditbeat/ so that if changes are needed in the future to best support the getting started experience we don't change the default file.

Reply:
I think we should have an only copy of these files, or the documentation references the other file, or packaging uses this one.

Reply:
Well, actually for example for metricbeat they are not the same, ok then.

Reply:
As I get familiar with auditbeat in Docker I may submit a PR with changes that will give a good getting started experience when managing containers.  I had started out using the default configs but then (as you saw with metricbeat) I wanted them changed.  Thanks for reviewing!